src/services/access.service.ts

```diff
@@ -95,9 +111,8 @@ async function createIndividualAccount(
     clientData,
   }
   const endpoint = constants.endpoints.bridgeAccount.replace(':id', '')
-  const result = await Utility.callEndpoint<any>(endpoint, 'POST', postData, token)
-
-  return result.ok
+  const result = await Utility.callEndpoint<IdentifierHolder>(endpoint, 'POST', postData, token)
+  return result.data
```

Comment:
Return the newly created user's identifier, so that the request reset password email can be sent to them